### DISCUSSION TITLE
"getActivePinia()" was called but there was no active Pinia.

### DISCUSSION THREAD

**Discussion by jadsow** [vuejs/pinia#2423] [Help and Questions]
"getActivePinia()" was called but there was no active Pinia.
I really need help to this.

My index store:
![image](https://github.com/vuejs/pinia/assets/95438480/c3d54524-0ecd-4203-9a4e-982d667ce2c6)

My store:
![image](https://github.com/vuejs/pinia/assets/95438480/4ad68906-8a29-4692-8f5f-a944833fe6a4)

My component:
![image](https://github.com/vuejs/pinia/assets/95438480/9983f82f-9cb4-43e3-bcb8-f529639ad618)

Error:
![image](https://github.com/vuejs/pinia/assets/95438480/4234db82-27b3-4439-8d4b-3e9c0f793ec4)

Can anybody help me please?

**Comment by angelhdzmultimedia** [comment]
Execute `npm add pinia`. It's not installed. Happened to me once, got that error, and when I checked, it was that when I installed pinia it didn't actually get installed, and I assumed it was installed. Hope that solves it because I can't think of anything else because don't have enough context. Your stores/index.js seems ok, and your stores.

**Reply by angelhdzmultimedia** [reply]
It also happens when we try to call a store before the pinia gets installed in Vue as a plugin.
